Source/Core/OrientedBoundingBox.js

```diff
@@ -419,6 +419,42 @@ define([
         return result;
     };
 
+    var scratchMatrix3PP = new Matrix3();
+    var scratchCartesian3PP = new Cartesian3();
+
+    /**
+     * If the given position is not already within the box, projects the given position onto the box.
+     * @param {Cartesian3} position The position being projected onto this OrientedBoundingBox.
+     * @returns {Cartesian3} A projected version of the inputted position if it was not originally within the OrientedBoundingBox.
+     */
+    OrientedBoundingBox.prototype.clampToBounds = function(position, result) {
+        //>>includeStart('debug', pragmas.debug);
+        Check.defined('position', position);
```

Comment:
Use `Check.typeof.Object`

Reply:
when i do that i claims the position is type of string thought it shows the [object object] statement for type of position - ideas?

Reply:
Check the order of your parameters - I think you have them swapped.